### PR TITLE
add blog

### PR DESCRIPTION
**Pull request by CalebProvost**
Add Android Screen Sharing Blog.

https://raw.githack.com/treehouses/treehouses.github.io/mobilescreenblog/#!pages/blog/20190925-mobilescreenshare.md

### CODE REVIEW

**Comment by pattanawadee88** [comment]
everything looks good for me :)

File: pages/blog/20190925-mobilescreenshare.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+<p style="width: 800px; heigth: auto;">![](images/20190925-Android-Settings.png)![](images/20190925-Android-About-Phone.png)![](images/20190925-Android-Build-Num.png)</p>
+
+#### Enable USB Debugging
+1. Now that you have developer mode unlocked, go to newly aquired developer settings (within the main settings page).
+1. Scroll down to "USB Debugging" and enable the option.
+<p style="width: 500px; heigth: auto;">![](images/20190925-Android-Developer.png)![](images/20190925-Android-USB-Debug.png)</p>
+
+## Use scrcpy!
+1. Connect your android device to your PC.
+1. A pop-up will ask for confirmation to allow connection with your PC, accept it.<p style="width: 300px; heigth: auto;">![](images/20190925-Android-Allow-Debug.png)</p>
```
Comment (dogi) [inline pages/blog/20190925-mobilescreenshare.md]: @Kid243 can you tell us how you did it in your blog

**Comment by Liew211** [comment]
@CalebProvost I just made a few small grammar/spelling fixes.

Most of the html tags in this shouldn't be necessary, especially for the `Notes` or alert boxes; the phone screenshot alignments are nice though.

**Comment by CalebProvost** [comment]
@Liew211 

> @CalebProvost I just made a few small grammar/spelling fixes.
> 
> Most of the html tags in this shouldn't be necessary, especially for the `Notes` or alert boxes; the phone screenshot alignments are nice though.

I disagree. Granted, the images could do without, but then you'll face some issues when on smaller devices (I was seeking to make it more responsive). This doesn't work fully so I'm debating on removing it all together and just resize the source of the images smaller.

As for the notes. The one's with the html are definitely needed as having a keyword cause the visual rendering does not work when a second one is supposed to be rendered within it. Especially not while keeping the count of the ordered list. I wasn't able to find how to at least.

So to have the code-block actually show up in the rendered note element block, I had to manipulate some html into the markdown.

**Review by dogi — APPROVED** [review]

**Pull request merged**